rpcd/playbooks/roles/rpc_maas/templates/openrc

```diff
@@ -4,7 +4,7 @@ export OS_ENDPOINT_TYPE={{ openrc_os_endpoint_type }}
 export OS_USERNAME={{ maas_keystone_user }}
 export OS_PASSWORD={{ maas_keystone_password }}
 export OS_TENANT_NAME={{ keystone_admin_tenant_name }}
-export OS_AUTH_URL={{ keystone_service_internalurl_v3 }}
+export OS_AUTH_URL={{ openrc_os_auth_url }}
```

Comment:
I seemed to have missed that when looking through variable name changes for the upgrade process.

Does this need to be renamed in the process of an upgrade?